.rubocop.yml

```diff
@@ -227,10 +228,10 @@ Layout/LeadingEmptyLines:
   Enabled: true
 
 Layout/LineLength:
-  Description: Limit lines to 100 characters.
+  Description: Limit lines to 120 characters.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#80-character-limits
   Enabled: true
-  Max: 100
+  Max: 120
```

Comment:
I am happy enough with 100. But I wanted to point out we do have our JS files with a matching 100, if we change this I think we should change them together to keep the whole codebase consistent

https://github.com/18F/identity-idp/blob/74f0ff081fb96eb02bb4f2665a01c7648f114f11/.prettierrc#L4